doc-content/enterprise-only/integration/codeready-studio-installing-runtime-environments-proc.adoc

```diff
@@ -1,24 +1,31 @@
 [id='codeready-studio-installing-runtime-environments-proc']
 = Installing {PRODUCT} runtime environments
 
-In order to create {PRODUCT} projects in Red Hat CodeReady Studio, it is necessary to install the {PRODUCT} runtime environment.
+In order to create {PRODUCT} projects in {CODEREADY_STUDIO}, it is necessary to install the {PRODUCT} runtime environment.
 
 A runtime environment is a collection of JAR files that represent a specific release of the software and includes the required libraries for compiling and running your business asset.
 
 .Prerequisites
-* Red Hat CodeReady Studio is installed.
+* {CODEREADY_STUDIO} is installed.
 
 .Procedure
 . Download the {DECISION_ENGINE}:
 .. Log in to the https://access.redhat.com[Red Hat Customer Portal].
 .. Click *DOWNLOADS* at the top of the page.
 .. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *{PRODUCT}*.
 .. On the *Software Downloads* page, download *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons* (`{PRODUCT_FILE}-add-ons.zip`).
-.. Extract `{PRODUCT_FILE}-add-ons.zip` and then extract the Drools runtime environment JAR files located in `{PRODUCT_FILE}-add-ons/{PRODUCT_FILE}-{URL_COMPONENT_DECISION_ENGINE}.zip`.
-. From the Red Hat CodeReady Studio menu, click *Window* -> *Preferences*.
+.. Extract `{PRODUCT_FILE}-add-ons.zip` and then extract the Drools runtime environment JAR files located in
```

Comment:
Drools is correct for DM and PAM?

Reply:
Checking with QE. Thank you.

Reply:
I confirmed both files in the CR1 RHDM and RHPAM builds.